modules/network/src/integrationTest/java/org/apache/ignite/network/scalecube/ITScaleCubeNetworkMessagingTest.java

```diff
@@ -106,7 +106,7 @@ public void messageWasSentToAllMembersSuccessfully() throws Exception {
             alice.messagingService().weakSend(member, testMessage);
 
         boolean messagesReceived = messageReceivedLatch.await(3, TimeUnit.SECONDS);
-        assertTrue(messagesReceived);
+        assertTrue(null);
```

Comment:
This is obviously an error

Reply:
This was made intentionally for testing logs on TC, will be reverted soon